mangaki/mangaki/utils/als.py

```diff
@@ -62,13 +64,15 @@ def factorize(self, matrix, random_state):
                 self.fit_work(work, matrixT)
 
     def fit(self, X, y):
-        print("Computing M: (%i × %i)" % (self.nb_users, self.nb_works))
+        if self.verbose:
+            print("Computing M: (%i × %i)" % (self.nb_users, self.nb_works))
```

Comment:
Let's make an issue. Because I'm myself confused because of `verbose`, `chrono` and `logging`.